test/e2e/specs/fakerp/command.go

```diff
@@ -38,6 +38,9 @@ var _ = Describe("Command tests [Command][Fake][LongRunning]", func() {
 		err := azurecli.OpenShiftManagedClustersAdmin.RunCommand(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"), vm, "RestartKubelet")
```

Comment:
you might want to rename the It block's description for the test from:

```
It("should be possible for an SRE to restart Kubelet and NetworkManager", func() {
```

to 

```
It("should be possible for an SRE to restart certain system services on VMs", func() {
```